package.json

```diff
@@ -365,7 +367,7 @@
     "react-window": "^1.8.5",
     "sass-loader": "^8.0.2",
     "sharp": "^0.26.1",
-    "spectron": "11.0.0",
+    "spectron": "14",
```

Comment:
Seems that we don't need spectron anymore, probably should be removed from here?